test/travis_test/build_2_container.sh

```diff
@@ -42,8 +42,14 @@ if [ "$1" == "compile_and_run_unit_test" ]; then
 
 elif [ "$1" == "2_port_test_traffic" ]; then
   echo "    --- 2_ports_CREATE test case---"
+  parent_container_ip=$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' aca_PARENT)
+  child_container_ip=$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' aca_CHILD)
+  echo "aca_PARENT ip: $parent_container_ip"
+  echo "aca_CHILD ip: $child_container_ip"
+  docker exec aca_CHILD -c "/etc/init.d/openvswitch-switch restart && ovs-vswitchd --pidfile --detach" || true
+  docker exec aca_PARENT -c "/etc/init.d/openvswitch-switch restart && ovs-vswitchd --pidfile --detach" || true
```

Comment:
this is done in line 26 already, do we need to do it again here?

Reply:
Removed these redundant lines and tested locally, the tests passed. Pushed the modified code. Please wait for the CI result. Thanks.